timesketch/lib/analyzers/crash.py

```diff
@@ -0,0 +1,178 @@
+"""Sketch analyzer plugin for crashes."""
+from __future__ import unicode_literals
+
+import re
+
+from timesketch.lib import emojis
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
+
+class CrashesSketchPlugin(interface.BaseSketchAnalyzer):
+    """Sketch analyzer for Windows Application Crashes."""
+
+    NAME = 'crashes'
+
+    DEPENDENCIES = frozenset()
+
+    def __init__(self, index_name, sketch_id):
+        """Initialize The Sketch Analyzer.
+
+        Args:
+            index_name: Elasticsearch index name
+            sketch_id: Sketch ID
+        """
+        self.index_name = index_name
+        super(CrashesSketchPlugin, self).__init__(index_name, sketch_id)
+
+    def run(self):
+        """Entry point for the analyzer.
+
+        Returns:
+            String with summary of the analyzer result
+        """
+        query_elements = {
```

Comment:
I'm wondering whether it makes more sense, or makes it a bit easier to change this file in the future to have this as a class constant

QUERY_ELEMENTS = ....

and thus you don't have to edit anything int eh run function, that can just parse this structure